### PR TITLE
[EOC-266] Deleting list

### PR DESCRIPTION
**Pull request by adamklepacz**

### CODE REVIEW

File: server/sockets/list.js [hunk 1]
```diff
@@ -606,6 +606,31 @@ const archiveList = (socket, dashboardClients, cohortClients, listClients) =>
...
+
+    dashboardClients.forEach(client => {
+      const { socketId } = client;
+
+      socket.broadcast
+        .to(socketId)
+        .emit(ListActionTypes.DELETE_SUCCESS, listId);
+    });
+
+    cohortClients.forEach(client => {
```
Comment (alemikolo) [inline server/sockets/list.js]: this should be done only when this is cohort's list so
```
if (cohortId) {}
```

Comment (alemikolo) [inline server/sockets/list.js]: You iterate through all cohortClients and send this event to all regardless of whether they are members of this list or not. You should send this event to list viewers which are on this cohort view

File: server/sockets/list.js [hunk 2]
```diff
@@ -606,6 +606,31 @@ const archiveList = (socket, dashboardClients, cohortClients, listClients) =>
...
+      const { socketId } = client;
+
+      socket.broadcast
```
Comment (alemikolo) [inline server/sockets/list.js]: Check if the viewer is on this cohort view (viewId === cohortId)

File: client/src/sockets/enums.js [hunk 3]
```diff
@@ -35,6 +35,7 @@ export const ListEvents = Object.freeze({
   ADD_VIEWER_SUCCESS: ListActionTypes.ADD_VIEWER_SUCCESS,
   ARCHIVE_SUCCESS: ListActionTypes.ARCHIVE_SUCCESS,
   CHANGE_TYPE_SUCCESS: ListActionTypes.CHANGE_TYPE_SUCCESS,
+  DELETE_AND_REDIRECT: 'list/DELETE_AND_REDIRECT',
```
Comment (marekrozmus) [inline client/src/sockets/enums.js]: why not in `ListActionTypes`?

Comment (adamklepacz) [reply client/src/sockets/enums.js]: beacuse this is specific to socket and we don't need action type for this.
The same approach is in the line 43 and 45 in this file. 
Check this out

**Pull request merged**